Tests/NIOHTTP1Tests/HTTPServerUpgradeTests.swift

```diff
@@ -18,7 +18,7 @@ import Dispatch
 @testable import NIOHTTP1
 
 extension ChannelPipeline {
-    func assertDoesNotContainUpgrader() throws {
+    fileprivate func assertDoesNotContainUpgrader() throws {
```

Comment:
Why did these become file private?

Reply:
They are tied to checking for 'HTTPServerUpgradeHandler'. I am currently writing a client handler, so they become a bit misleading. I could have renamed them, but I thought it was just easier to make them fileprivate as that is where they are used. 

To be honest, it isn't a big deal, I was more concerned with the other changes. I can see that the general pattern seems to be to put extensions for testing in the most relevant file but allow usage from any other test file. If you would rather not affect that, I can make separate client versions that explicitly have the word 'client', for example, and leave these alone.

Reply:
No, your reasoning makes sense, I just wanted to understand the change. For now making these `fileprivate` shouldn't hurt, we can always deal with the issue later if we need them elsewhere.